src/lib/components/table/table-settings/dialog/table-settings-dialog.glob.scss

```diff
@@ -1,6 +1,7 @@
 mat-dialog-container {
     &.mat-dialog-container {
         .mat-dialog-content {
+            background-color: var(--color-structure-0);
```

Comment:
Since this is a global stylesheet and the selector is `.mat-dialog-content` it would apply to all dialogs!

I don't think this is intended. You should scope this style properly.

Reply:
You may actually remove this stylesheet from the global styles and encapsulate it with the component instead.
But be careful, I think a few styled still need to bleed through the scope of this component (if these are still necessary).